lib/index.js

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const _ = require('lodash');
+const bedrock = require('bedrock');
+const {BedrockError} = bedrock.util;
+
+const api = {};
+module.exports = api;
+
+/**
+ * Dereferences the service descriptors for the electors listed in electorPool.
+ *
+ * @param {Object[]} electorPool an array of electors.
+ * @param {Object} ledgerNode a LedgerNode instance.
+ *
+ * @return {Promise<Object>} the service endpoints for the electors.
+ * @throws if there is difficulty dereferencing any service descriptors.
+ */
+api.dereferenceElectorPool = async ({electorPool, ledgerNode}) => {
+  const electors = {};
+
+  // FIXME: is type an array or a string?  Value is TBD
+  const continuityServiceType = 'Continuity2017Peer';
+
+  for(const e of electorPool) {
+    // service may be a string referencing a serviceId contained in a DID
+    // Document referenced in `e.elector`
+    if(typeof e.service === 'string') {
+      // dereference elector's DID document to locate the service descriptor
+      // `records.get` throws NotFoundError on an unknown recordId
+      const {record} = await ledgerNode.records.get({recordId: e.elector});
```

Comment:
Seems like we'd need this locked to a particular `blockHeight` for it to function properly, right?

Reply:
Same applies here, don't we always want the latest revision, which by definition cannot be ahead of the block that is currently be computed.